translations/locales/ja/translations.json

```
@@ -358,7 +358,8 @@
       "Description": "ユーザー登録",
       "Or": "もしくは",
       "AlreadyHave": "既にアカウントをお持ちですか？",
-      "Login": "ログイン"
+      "Login": "ログイン",
+      "Warning": "サインアップすると、p5.js エディターの<0>利用規約</0> と <1>プライバシー ポリシーに</1> 同意したことになります。"
```

```suggestion
      "Warning": "アカウント作成をすると、p5.js エディターの <0>利用規約</0> と <1>プライバシー ポリシー</1> に同意したことになります。"
```

It would be better to match the translation of `SubmitSignup`.

---

> It would be better to match the translation of `SubmitSignup`.

thanks for pointing out , although i made confirm the text with my 'nihongo-friend' i didn';t well placed the tags.
arigatou: )